admin_board_view/templates/user_home.html

```diff
@@ -91,6 +91,7 @@ <h5>Top up balance</h5>
                 <button type="submit" class="btn btn-primary">Top up</button>
               </div>
             </div>
+            <small id="feeHelp" class="form-text text-muted">Plus &euro;0.39 transaction fee.</small>
```

Comment:
This should be configurable, the 0.39

Reply:
oh yeah you're right, I missed that one!